packages/webdriverio/src/index.ts

```diff
@@ -53,6 +52,7 @@ export const remote = async function (params: RemoteOptions, remoteModifier?: Fu
     const ProtocolDriver = require(automationProtocol).default
 
     params = Object.assign({}, detectBackend(params), params)
+    params.automationProtocol = automationProtocol
```

Comment:
the `automationProtocol` is already passed to `updateCapabilities` in the line below it seems. is this actually needed?

Reply:
yes, we want to explicitly set it to ensure that when attaching to it ends up attaching to the correct protocol. But good point on `updateCapabilities`. I moved the assignment in there.